src/Features/CSharp/Portable/GenerateMember/GenerateParameterizedMember/CSharpGenerateParameterizedMemberService.cs

```diff
@@ -41,13 +41,9 @@ protected override ITypeSymbol DetermineReturnTypeWorker(CancellationToken cance
                 // Defer to the type inferrer to figure out what the return type of this new method
                 // should be.
                 var typeInference = this.Document.Project.LanguageServices.GetService<ITypeInferenceService>();
-                var inferredType = typeInference.InferType(this.Document.SemanticModel,
-                    _invocationExpression, objectAsDefault: true, cancellationToken: cancellationToken);
-                if (State.IsInConditionalAccessExpression)
```

Comment:
>State.IsInConditionalAccessExpression [](start = 20, length = 37)

I don't see us doing this filtering in the type inference service itself but I may have missed it.  Are we still changing the inferred type if it is nullable?